pkg/extensions/tbtc/tbtc.go

```diff
@@ -439,6 +440,17 @@ func (t *tbtc) monitorAndAct(
 			return
 		}
 
+		if !t.monitoringLock.tryLock(depositAddress, monitoringName) {
+			logger.Warningf(
+				"lock for [%v] monitoring for deposit [%v] has been "+
+					"already acquired; could not start the monitoring",
```

Comment:
I'd rather say the given deposit is already monitored for this event. If I were reading logs and saw a message about already acquired lock and monitoring that could not be started because of that, I'd think it's something wrong with synchronization. While here, it's just a normal thing with duplicate events.